scripts/processing/09-run_time_decoding.py

```diff
@@ -51,7 +51,8 @@ def run_time_decoding(subject_id, condition1, condition2):
     epochs.pick_types(meg='mag').pick_channels(ch_names)
 
     # Use AUC because chance level is same regardless of the class balance
-    td = TimeDecoding(predict_mode='cross-validation', scorer='roc_auc')
+    td = TimeDecoding(predict_mode='cross-validation', scorer='roc_auc',
+                      n_jobs=N_JOBS)
```

Comment:
hmm ... why did you change this? To save memory? Perhaps worth adding a comment because it breaks the pattern we use in our other scripts

Reply:
Yes it saves memory and should be roughly as fast